xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-macro/macroWizard.js

```diff
@@ -25,6 +25,7 @@ define('macroWizard', ['macroSelector', 'macroEditor'], function(selectMacro, ed
       data.macroId = data.macroCall && data.macroCall.name;
       return insertMacroWizard(data);
     } else {
+      delete data.macroCall.parameters.$content;
```

Comment:
I'd add a comment explaining that the macro content is kept in a different field. Here you're just removing a temporary field used by the macro editor. Couldn't this be done in the macro editor?

Reply:
> Couldn't this be done in the macro editor?

It probably could but it felt safer for me to do it there as I'm sure it's the before returning the value and after all operations are done in macroEditor. I agree for the comment I just forgot.